### PR TITLE
Chapter 9 - Remember me

### PR DESCRIPTION
**Pull request by htpt2910**
image

### CODE REVIEW

**Comment by htpt2910** [comment]
ready

**Comment by htpt2910** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -21,6 +23,25 @@ def digest string
...
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute(:remember_digest, User.digest(remember_token))
+  end
+
+  def authenticated? remember_token
+    return false if remember_digest.nil?
```
Comment (manhnt-0960) [inline app/models/user.rb]:
```suggestion
    return false unless remember_digest
```

**Review by manhnt-0960 — COMMENTED** [review]
fix rồi merged

**Pull request merged**